docs/resources/Guild.md

```diff
@@ -1088,6 +1089,44 @@ Returns a PNG image widget for the guild. Requires no permissions or authenticat
 | banner3 | large image with guild icon, name and online count. In the footer, Discord logo on the left and "Chat Now" on the right                                        | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner3) |
 | banner4 | large Discord logo at the top of the widget. Guild icon, name and online count in the middle portion of the widget and a "JOIN MY SERVER" button at the bottom | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner4) |
 
+## Get Guild Discovery Metadata % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/discovery-metadata
+
+Returns the [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) object for the guild. Requires the `MANAGE_GUILD` permission.
+
+## Modify Guild Discovery Metadata % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/discovery-metadata
+
+Modify the [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) for the guild. Requires the `MANAGE_GUILD` permission. Returns the updated [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) object on success.
+
+> info
+> All parameters to this endpoint are optional and nullable. Omitting or setting a null value will set it to default.
+
+###### JSON Params
+
+| Field                         | Type                                                                         | Description                                                                                    | Default |
+| ----------------------------- | ---------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------- |
+| primary_category_id           | [discovery category](#DOCS_RESOURCES_DISCOVERY/discovery-category-object) id | the id of the primary discovery category                                                       | 0       |
+| keywords                      | array of strings                                                             | up to 10 discovery search keywords                                                             | null    |
+| emoji_discoverability_enabled | boolean                                                                      | whether guild info should be shown when custom emojis and stickers from this guild are clicked | true    |
+
+## Add Guild Discovery Subcategory % POST /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/discovery-categories/{category.id#DOCS_RESOURCES_DISCOVERY/discovery-category-object}
```

Comment:
this should be PUT, as POST is not supported on api v10 and up

Reply:
ah I didn't notice that, updated - looks like it also 204s now on both PUT and pre-v10 POST